run_benchmark_multi.sh

```diff
@@ -16,15 +16,14 @@ hash=$(git rev-parse HEAD)
 make build-headless
```

Comment:
please make sure this script complied wit the [shell style guide](https://google.github.io/styleguide/shell.xml)